app/components/Chatbot/index.tsx

```diff
@@ -248,7 +249,7 @@ export const Chatbot = ({question, questions}: {question?: string; questions?: s
   })
 
   return (
-    <div className="centered col-9 padding-bottom-128">
+    <div className="centered col-9 height-70">
```

Comment:
I believe this should take up 10 columns
<img width="946" alt="image" src="https://github.com/StampyAI/stampy-ui/assets/825910/cfbb088e-2d8a-4067-ae4d-44bf97395aaf">

Reply:
If you want to leave this to me when I do css that's totally fine